pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -490,7 +510,11 @@ func (s *DiscoveryServer) edsUpdate(clusterID, serviceName string, namespace str
 
 	// 2. Update data for the specific cluster. Each cluster gets independent
 	// updates containing the full list of endpoints for the service in that cluster.
+	mtlsReady := true //TODO GregHanson handle 0 endpoint case
```

Comment:
you no longer need to gather `whether all endpoints of a service have mTLSReady = true`. This information is no longer helpful for us. I think. 

/cc @rshriram

Reply:
removed service level mtls logic per networking workgroup discussion on today's call